build.gradle

```diff
@@ -202,7 +203,7 @@ dependencies {
   annotationProcessor group:'org.mapstruct', name: 'mapstruct-processor', version: versions.mapstruct
   annotationProcessor group: 'org.projectlombok', name: 'lombok', version: versions.lombok
 
-  runtime group: 'org.postgresql', name: 'postgresql', version: '42.2.19'
+  runtime group: 'org.postgresql', name: 'postgresql', version: '.Values.postgresql.postgresqlDatabase'
```

Comment:
Is this a typo?

Reply:
Yes, will fix

Reply:
Fixed in https://github.com/hmcts/cpo-case-payment-orders-api/pull/16/commits/caada6f64dfaa5a02352eb768ee9792564c98ff1